functions.php

```diff
@@ -201,7 +201,7 @@ function twentysixteen_fonts_url() {
  * @since Twenty Sixteen 1.0
  */
 function twentysixteen_javascript_detection() {
-	echo "<script>(function(html){html.className = html.className.replace(/\bno-js\b/,'js')})(document.documentElement);</script>\n";
+	echo "<script>(function(html){html.className = html.className.replace(/\bno-js\b/,'') + ' js'})(document.documentElement);</script>\n";
```

Comment:
That's correct, aim is to add the `js` class is independent of the `no-js` class. I'm not sure what you mean by it not feeling right, can you expand? 

The code would be clearer as multiple assignments but I'm trying avoid multiple DOM repaints.